Sources/MapboxMaps/Foundation/Camera/CameraManager.swift

```diff
@@ -362,13 +362,16 @@ public class CameraManager {
     ///   - duration: Duration of the animation, measured in seconds. If nil, a suitable calculated duration is used.
     ///   - completion: Completion handler called when the animation stops
     /// - Returns: The optional `CameraAnimator` that will execute the FlyTo animation
-    public func flyTo(to camera: CameraOptions,
+    public func fly(to camera: CameraOptions,
```

Comment:
why are we renaming to Fly? I believe android calls theirs FlyTo

Reply:
This is also `fly(to:)` which is semantically the same thing.

Reply:
Fair, yeah that makes sense. Will update mobile metrics